sdk/textanalytics/azure-ai-textanalytics/README.md

```diff
@@ -446,7 +446,7 @@ and [language and regional support][language_and_regional_support].
 
```

Comment:
I can't comment on it, but in the https://github.com/Azure/azure-sdk-for-python/blob/f7322881efba84d2f64df15f3370844b64e34942/sdk/textanalytics/azure-ai-textanalytics/README.md#recognize-pii-entities, would it be better to say ` This endpoint is only available for v3.1` like you have in other places?

Reply:
Yes, will change. I always forget that PII wasn't added until 3.1.